crates/vizia_core/src/context/backend.rs

```diff
@@ -264,4 +265,19 @@ impl<'a> BackendContext<'a> {
     pub fn needs_refresh(&mut self) {
         self.0.style.system_flags = SystemFlags::all();
     }
+
+    pub fn emit_scheduled_events(&mut self) {
```

Comment:
Shouldnt this be called like "wait_for_scheduled_events"?

Reply:
Sure but you're also emitting those events by moving them to the event queue.